catroid/src/main/java/org/catrobat/catroid/common/BrickValues.java

```diff
@@ -131,9 +131,9 @@
 	public static final double ADD_ITEM_TO_USERLIST = 1d;
 	public static final int DELETE_ITEM_OF_USERLIST = 1;
 	public static final int INSERT_ITEM_INTO_USERLIST_INDEX = 1;
-	public static final double INSERT_ITEM_INTO_USERLIST_VALUE = 1d;
+	public static final double INSERT_ITEM_INTO_USERLIST_VALUE = 0.0;
 	public static final int REPLACE_ITEM_IN_USERLIST_INDEX = 1;
-	public static final double REPLACE_ITEM_IN_USERLIST_VALUE = 1d;
+	public static final double REPLACE_ITEM_IN_USERLIST_VALUE = 0.0;
```

Comment:
Why these changes?

Reply:
i just pushed it so i have it on my second computer to continue fixxing the espresso test.

The reason was simply to check why the behaviour was different for different bricks which both had double as default value. Now I already found out why InsertIntoUserListBrick and ReplaceItemInUserlistBrick default value tests failed. It seems since i now have two input fields with the same default value the method checkIfBrickShowsText(...) goes into an exception, since it finds two fields which show the same text. I'll fix this ticket tomorrow and then this line should be back to normal.

Reply:
Ah, got it!